examples/scale/scenarios/basic_scenario.go

```diff
@@ -0,0 +1,23 @@
+package scenarios
+
+import (
+	"fmt"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+// BasicScenario is a struct with Scenario embedded that implements mmf.Run and evaluator.Evaluate methods.
```

Comment:
This comment is repeating exactly what the code is saying....

Reply:
This comment is added to make the golangci bot happy. Any better suggestions?

Reply:
package private.